tracer/src/Datadog.Trace.Tools.Runner/Checks/ProcessBasicCheck.cs

```diff
@@ -460,5 +497,169 @@ private static bool ParseBooleanConfigurationValue(string value)
                 or "y"
                 or "1";
         }
+
+        private static bool TracingWithBundle(string? profilerPathValue)
+        {
+            if (profilerPathValue is null)
+            {
+                return false;
+            }
+
+            string[] expectedEndingsForBundleSetup =
+            {
+                "/datadog/linux-musl-x64/Datadog.Trace.ClrProfiler.Native.so",
+                "/datadog/linux-x64/Datadog.Trace.ClrProfiler.Native.so",
+                "/datadog/linux-arm64/Datadog.Trace.ClrProfiler.Native.so",
+                "\\datadog\\win-x64\\Datadog.Trace.ClrProfiler.Native.dll",
+                "\\datadog\\win-x86\\Datadog.Trace.ClrProfiler.Native.dll"
+            };
+
+            foreach (var bundleSetupEnding in expectedEndingsForBundleSetup)
+            {
+                if (profilerPathValue.EndsWith(bundleSetupEnding, StringComparison.OrdinalIgnoreCase))
+                {
+                    return true;
+                }
+            }
+
+            return false;
+        }
+
+        private static void NoManageCodeClrVersionIis(ProcessInfo process)
+        {
+            foreach (var module in process.Modules)
+            {
+                var fileName = Path.GetFileName(module);
+
+                if (fileName.Equals("Microsoft.AspNetCore.Server.IIS.dll", StringComparison.OrdinalIgnoreCase))
+                {
+                    AnsiConsole.WriteLine(TracingDotnetOnIis);
+                }
+            }
+        }
+
+        private static string? CheckWindowsInstallation()
+        {
+            string? tracerVersion = null;
+            string? displayName = null;
+
+            if (RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
+            {
+                const string datadog64BitProgram = "Datadog .NET Tracer 64-bit";
+                const string datadog32BitProgram = "Datadog .NET Tracer 32-bit";
+
+                var registrySubKeys = new[] { @"SOFTWARE\Microsoft\Windows\CurrentVersion\Uninstall", @"SOFTWARE\WOW6432Node\Microsoft\Windows\CurrentVersion\Uninstall" };
+
+                foreach (var registryKey in registrySubKeys)
+                {
+                    using (var key = Registry.LocalMachine.OpenSubKey(registryKey))
+                    {
+                        if (key is not null)
+                        {
+                            foreach (var subKeyName in key.GetSubKeyNames())
+                            {
+                                var subKey = key.OpenSubKey(subKeyName);
+
+                                if (subKey?.GetValue("DisplayName") is datadog64BitProgram or datadog32BitProgram && displayName is null)
+                                {
+                                    displayName = (string?)subKey.GetValue("DisplayName");
+                                    tracerVersion = subKey.GetValue("VersionMajor") + "." + subKey.GetValue("VersionMinor");
+                                    break;
```

Comment:
This break will only exit from the inner foreach loop. If you want to exit the outer foreach loop you'd need to add another `break` instruction with some logic